app/services/report_errors_to_argo.rb

```diff
@@ -0,0 +1,24 @@
+class ReportErrorsToArgo
+  WORKFLOW = 'preservationWF'.freeze
+  WORKFLOW_DEF = <<-EOXML
+    <workflow id="preservationWF">
+         <process name="start" />
+         <process name="moab-valid"  />
+         <process name="M2C-check-exist" />
+         <process name="C2M-check-exist" />
+    </workflow>
+    EOXML
+  REPO = "dor".freeze
```

Comment:
I *think* the pattern is to put this stuff in config -- see https://github.com/sul-dlss/was_robot_suite/blob/9fa3d924c62d5b496a305d258ac15c1c9a9731a2/config/workflows/dor/wasCrawlDisseminationWF.xml for example ...